src/__tests__/auto-injectable.test.ts

```diff
@@ -1,15 +1,15 @@
-import {autoInjectable, injectable} from "../decorators";
-import {instance as globalContainer} from "../dependency-container";
+import { autoInjectable, injectable } from "../decorators";
```

Comment:
Is there a reason that you're adding in the space inside the braces? I think the TSLint rules we had didn't like that (but I could be wrong).

Reply:
@MeltingMosaic, yeah, prettier did some weird things (probably because I dont have a `.prettierrc` yet so its using default config).